### PR TITLE
normalize_loadshapes

### PR DESCRIPTION
**Pull request by berkserbet**
### Why:
* Normalization of loadshapes

### What:
* Normalizes load shapes based on summer peak total

### Checklist
- [x] Finished My Changes
- [ ] Automated Unit Tests

### CODE REVIEW

File: pipelines/rbsa/tasks/normalize_loadshapes.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+            self.enduse_cols = list(city_df.columns)
+            self.enduse_cols.remove('target')
+            self.enduse_cols.remove('daytype')
+
+            # get enduse columns
+            normalization_val = self.get_normalization_val(city_df, summer=True)
+
+            # Normalize by peak total
+            city_df[self.enduse_cols] = city_df[self.enduse_cols]/normalization_val
```
Comment (jongoncalves) [inline pipelines/rbsa/tasks/normalize_loadshapes.py]: whitespace around `/`

File: pipelines/rbsa/tasks/normalize_loadshapes.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            if initialization:
+                normal_loadshapes = city_df
+                initialization = False
+            else:
+                normal_loadshapes = normal_loadshapes.append(city_df)
+
+        self.validate(normal_loadshapes)
+        self.on_complete({self.output_artifact_normal_loadshapes: normal_loadshapes})
+
+    def get_normalization_val(self, city_df, summer):
```
Comment (jongoncalves) [inline pipelines/rbsa/tasks/normalize_loadshapes.py]: Seems like on line 48 summer is hardcoded to True, when is summer = False?
Can we simplify some of this code? 
What drives the change in the boolean for the summer value?

File: pipelines/rbsa/tasks/normalize_loadshapes.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        """        
+        if summer:
+            city_df = city_df.loc[city_df['daytype'] == 'summer_peak']
+
+        totals = city_df[self.enduse_cols].sum(axis=1)
+        normalization_val = totals.max() # can be adjusted if normalizing for summer peak
+
+        # can check here for totals.idxmax() month to confirm summer/winter
+
+        return normalization_val
```
Comment (jongoncalves) [inline pipelines/rbsa/tasks/normalize_loadshapes.py]: Remove line 71 and just return `totals.max()`

Also, put the comment in line 73 in the docstring

File: pipelines/rbsa/tasks/normalize_loadshapes.py (hunk 1 above)
Comment (berkserbet) [reply pipelines/rbsa/tasks/normalize_loadshapes.py]: fixed

File: pipelines/rbsa/tasks/normalize_loadshapes.py (hunk 2 above)
Comment (berkserbet) [reply pipelines/rbsa/tasks/normalize_loadshapes.py]: Summer = True will normalize to peak summer, and summer = False to peak at any time. I'm not sure if we will be asked to change this later on.

File: pipelines/rbsa/tasks/normalize_loadshapes.py (hunk 3 above)
Comment (berkserbet) [reply pipelines/rbsa/tasks/normalize_loadshapes.py]: fixed

File: pipelines/rbsa/tasks/normalize_loadshapes.py (hunk 2 above)
Comment (jongoncalves) [reply pipelines/rbsa/tasks/normalize_loadshapes.py]: I see - thanks for explanation. Let's come up with a way for this to be dynamically set in the future.

**Pull request merged**